indico/modules/events/papers/controllers/management.py

```diff
@@ -51,3 +54,32 @@ def _process(self):
             logger.info("Paper managers of %r updated by %r", self.event_new, session.user)
             return jsonify_data(flash=False)
         return jsonify_form(form)
+
+
+class RHSwitchReviewingType(RHManagePapersBase):
+
+    CSRF_ENABLED = True
+
+    def _process_PUT(self):
+        event = self.event_new
+        reviewing_type = request.view_args['reviewing_type']
+        if reviewing_type not in ('content', 'layout'):
+            raise Forbidden('Reviewing type not available')
+        event.cfp.toggle_reviewing(reviewing_type=reviewing_type, enable=True)
+        flash(_("Reviewing type enabled"), 'success')
+        logger.info("Reviewing type '{}' for event {} enabled by {}".format(reviewing_type, event, session.user))
+        event.log(EventLogRealm.management, EventLogKind.positive, 'Papers',
+                  "Enabled reviewing type '{}'".format(reviewing_type), session.user)
+        return jsonify_data(enabled=True)
+
+    def _process_DELETE(self):
+        event = self.event_new
+        reviewing_type = request.view_args['reviewing_type']
+        if reviewing_type not in ('content', 'layout'):
+            raise Forbidden('Reviewing type not available')
+        event.cfp.toggle_reviewing(reviewing_type=reviewing_type, enable=False)
```

Comment:
I think it'd make sense to create a `toggle_review_type` method in `operations.py`, but sure, I agree with you.